src/platform/plugins/shared/data/server/ui_settings.ts

```diff
@@ -271,8 +271,7 @@ export function getUiSettings(
     },
     [UI_SETTINGS.SEARCH_INCLUDE_FROZEN]: {
       name: 'Search in frozen indices',
-      description: `Will include <a href=${docLinks.links.elasticsearch.frozenIndices}
-        target="_blank" rel="noopener">frozen indices</a> in results if enabled. Searching through frozen indices
+      description: `Will include frozen indices in results if enabled. Searching through frozen indices
```

Comment:
I think this will be removed soon anyway, but I'm guessing we wouldn't want to keep a link to the old docs until then, would we?

Reply:
Thanks for confirming. I think we're good to remove it since we don't want to promote it, and the setting is going away soon anyway. cc @lukasolson for awareness just in case you have thoughts.